tests/functional/heketi/test_heketi_brick_evict.py

```diff
@@ -140,10 +148,11 @@ def test_brick_evict_with_node_down(self):
 
         # Get list of hostname from node id
         host_list = []
-        for node_id in node_list[3:]:
-            node_info = heketi_ops.heketi_node_info(
-                h_node, h_server, node_id, json=True)
-            host_list.append(node_info.get('hostnames').get('manage')[0])
+        if node_count == "three_nodes":
+            for node_id in node_list:
+                node_info = heketi_ops.heketi_node_info(
+                    h_node, h_server, node_id, json=True)
+                host_list.append(node_info.get('hostnames').get('manage')[0])
```

Comment:
Isn't this step is common for both three_nodes" and "more than three nodes"?
see line number 162

Reply:
No these are the nodes that are disabled.